site/content/docs/main/plugin-release-instructions.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Releasing Velero plugins
+layout: docs
+toc: "true"
+---
+
+# Releasing Velero plugins
+
+Velero plugins maintained by the core maintainers do not have any shipped binaries, only container images, so there is no need to invoke a GoReleaser script.
+Container images are built via a CI job on git push.
+
+1. Update the README.md file to update the compatibility matrix and `velero install` instructions with the expected version number and open a PR.
```

Comment:
I don't know if it fits into this document, but how do we determine what the new version of a plugin in should be? Also, what affects a plugins compatibility with particular velero versions?

Reply:
New versions are usually based on semantic versioning - honestly we may not actually need new releases for AWS and GCP for v1.5.x, but I need to look at what actually changed.

However, for the CSI plugin, the new version will be using API calls introduced in Velero v1.5.1 (https://github.com/vmware-tanzu/velero-plugin-for-csi/pull/70), so we'll need to make sure that's called out. Other compatibility concerns are things like the plugin interface itself changing with new methods being added, changed, or removed.